internal/service/databaseuser/data_source_database_user.go

```diff
@@ -55,8 +57,9 @@ func (d *databaseUserDS) Schema(ctx context.Context, req datasource.SchemaReques
 				Required: true,
 			},
 			"password": schema.StringAttribute{
-				Computed:  true,
-				Sensitive: true,
+				Computed:           true,
+				Sensitive:          true,
+				DeprecationMessage: fmt.Sprintf(constant.DeprecationParamByVersion, "1.16.0"),
```

Comment:
sure

Reply:
Thanks for creating it, would it be possible to add some more description? At minimum:
- the reference of this PR, 
- the fields to remove

otherwise if we do this in say two months we won't remember exactly what to do